docs/details/develop-checks.md

```diff
@@ -1,22 +1,184 @@
-# Develop Security Checks
-
-The Security Threat Tool in PMM offers a set of checks that can detect common security threats, performance degradation, data loss and data corruption.
-
-As a developer, you can create custom checks to cover additional use cases that are relevant to your specific database infrastructure.
-
+# Develop custom checks
```

Comment:
the topic describes how to develop checks for previous PMM versions as well. The introductory parts use "checks" instead of "advisor checks" or "security checks" to cover both scenarios

Reply:
I'm still not sure about the name. Maybe @rnovikovP can advise us.

Reply:
@rnovikovP, @PavelKhripkov, I think I've implemented all your feedback. 
I've also added some interactivity with collapsible and tabbed notes to improve the structure of the document. Could you please check the draft page here: 
https://pmm-doc-pr-746.onrender.com/.